discord/application.go

```diff
@@ -29,30 +31,51 @@ type Command struct {
 	Options []CommandOption `json:"options,omitempty"`
```

Comment:
```suggestion
	//
	// It is only present on ChatInputCommands.
	Options []CommandOption `json:"options,omitempty"`
```
Since `options` is now only valid for `ChatInputCommand`s, that should probably be noted in `Options` doc.